### PR TITLE
8298083: The "CheckBox/RadioButton[Enabled/Disabled].textForeground" stoped working

### PR DESCRIPTION
**Pull request by mrserb**
Hi all,
This pull request contains a backport of commit [5540a8c5](https://github.com/openjdk/jdk/commit/5540a8c5b7160ab5c67bb84631e3de54fa5aeceb) from the [openjdk/jdk](https://git.openjdk.org/jdk) repository.
The commit being backported was authored by Sergey Bylokhov on 8 Dec 2022 and was reviewed by Prasanta Sadhukhan.
Thanks!

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8298083](https://bugs.openjdk.org/browse/JDK-8298083): The "CheckBox/RadioButton[Enabled/Disabled].textForeground" stoped working

### Reviewers
 * [Phil Race](https://openjdk.org/census#prr) (@prrace - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk20 pull/17/head:pull/17` \
`$ git checkout pull/17`

Update a local copy of the PR: \
`$ git checkout pull/17` \
`$ git pull https://git.openjdk.org/jdk20 pull/17/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 17`

View PR using the GUI difftool: \
`$ git pr show -t 17`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk20/pull/17.diff">https://git.openjdk.org/jdk20/pull/17.diff</a>

</details>

### CODE REVIEW

**Comment by mrserb** [comment]
@psadhukhan please take a look

**Comment by prsadhuk** [comment]
I guess for clean backport doesn't need a reviewer..

**Comment by mrserb** [comment]
> I guess for clean backport doesn't need a reviewer.

@prrace can you please confirm that, so I can integrate

**Comment by prrace** [comment]
It requires reviewer approval
"No reviewer for a clean backport" is allowed for update release backports,
which do however require the UR maintainer to approve,
This isn't an update release. It is a feature release and there's no maintainer/lead approval required before RDP2.
This came up once before and skara marking the PR as "ready" is a bug.
Actually filing a skara bug hasn't happened but I am assured it will be done.

**Review by prrace — APPROVED** [review]
OK. I see this would have gone into 20 directly but it missed the fork by just a little bit,
and it is a backout, so approving.

**Pull request closed without merging**
